<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoloTitleWidgetClass</class>
 <widget class="QWidget" name="HoloTitleWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoloTitleWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="menuLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="minimizeButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover {    
        border:none;                                       
        image:url(:/Image/icon/Image/icon/dash-lg.png);   
}                      
QPushButton:hover {                                  
        image:url(:/Image/icon/Image/icon/dash-lg.png);   
}                                               
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="maximizeButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>13</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">    QPushButton:!hover{                                 
        border:none;                                    
        image:url(:/Image/icon/Image/icon/square.png)    
}                                                       
    QPushButton:hover {                                 
        border:none;                                    
        image:url(:/Image/icon/Image/icon/square-fill.png);   
}                                                       
    QPushButton:pressed {                               
        border:none;                                    
        image:url(:/Image/icon/Image/icon/square.png);   
} </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">    QPushButton:!hover{                                 
        border:none;                                    
        image:url(:/Image/icon/Image/icon/x-circle.png)    
}                                                       
    QPushButton:hover {                                 
        border:none;                                    
        image:url(:/Image/icon/Image/icon/x-circle-fill.png);   
}                                                       
    QPushButton:pressed {                               
        border:none;                                    
        image:url(:/Image/icon/Image/icon/x-circle.png);   
} </string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
